builtin/game/chatcommands.lua

```diff
@@ -677,6 +677,38 @@ core.register_chatcommand("spawnentity", {
 	end,
 })
 
+core.register_chatcommand("placenode", {
+	params = "<NodeName> [<X>,<Y>,<Z>]",
+	description = "Places node at given (or your) position",
+	privs = { give = true, interact = true, protection_bypass, server },
```

Comment:
```suggestion
	privs = {interact = true, server = true},
```

- You missed `= true`.
- Checking if the player has `protection_bypass` should be done within `func` and it shouldn't be a required priv (not that it really matters - the caller would have the `server` priv anyway).